eraser-map.yaml

```diff
@@ -919,7 +919,7 @@ layers:
         data: { source: osm, layer: water }
         draw:
             grid:
-                order: 5
+                order: 2
```

Comment:
Water must be on top of other landuse (else ponds and lakes inside of landuse areas disappear).

This needs to stay at `5` (but piers could move to say, `6`).